docker/files/usr/local/etc/gunicorn/marsha.py

```diff
@@ -9,6 +9,8 @@
 graceful_timeout = 90
 timeout = 90
 workers = 3
+worker_class = gthread
+threads = 6
```

Comment:
You should also set the [`worker_tmp_dir`](https://docs.gunicorn.org/en/stable/settings.html#worker-tmp-dir):

```python
worker_tmp_dir = "/dev/shm"
```